src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -395,13 +395,14 @@ private async Task PublishFunctionApp(Site functionApp, GitIgnoreParser ignorePa
             else if (functionApp.IsLinux && functionApp.IsElasticPremium)
             {
                 // Elastic Premium Linux
-                shouldSyncTriggers = await HandleElasticPremiumLinuxPublish(functionApp, zipStreamFactory);
+                shouldSyncTriggers = true;
```

Comment:
For premium, it's better to always sync trigger to notify the platform (remote build or not)